integration-cli/cli/cli.go

```diff
@@ -91,9 +118,9 @@ func appendDocker(cmd *icmd.Cmd) {
 	cmd.Command = append([]string{testEnv.DockerBinary()}, cmd.Command...)
 }
 
-// Cmd build an icmd.Cmd struct from the specified command and arguments
-func Cmd(command string, args ...string) icmd.Cmd {
-	return icmd.Command(command, args...)
+// Args build an icmd.Cmd struct from the specified arguments
+func Args(args ...string) icmd.Cmd {
```

Comment:
@cpuguy83 because its only for args actually (we prepend it with the `dockerBinary` so `command` did not make sense)